src/main/java/org/radarbase/output/source/StorageIndexManager.kt

```diff
@@ -0,0 +1,8 @@
+package org.radarbase.output.source
+
+interface StorageIndexManager {
```

Comment:
It might be good to add a line of documentation to these interfaces (i.e. StorageIndex, StorageNode, StorageIndexManager)